validator/validate.py

```diff
@@ -58,6 +57,10 @@ def validate(path, format='json',
                          overrides=overrides, for_appversions=for_appversions)
     bundle.save_resource('is_compat_test', compat_test)
 
+    if (approved_applications is None):
+        approved_applications = os.path.join(os.path.dirname(__file__),
```

Comment:
This looks like it doesn't pass PEP8 but I don't see a build failure. Maybe we aren't running PEP8 tests...

This should be:
```py
approved_applications = os.path.join(
    os.path.dirname(__file__),
    'app_versions.json')
# or
approved_applications = os.path.join(os.path.dirname(__file__),
                                     'app_versions.json')